charts/argo-cd/templates/_helpers.tpl

```diff
@@ -144,7 +144,9 @@ app.kubernetes.io/component: {{ .component }}
 Return the appropriate apiVersion for ingress
 */}}
 {{- define "argo-cd.ingress.apiVersion" -}}
-{{- if semverCompare "<1.14-0" (include "argo-cd.kubeVersion" $) -}}
+{{- if .Values.APIOverrides -}}
```

Comment:
```suggestion
{{- if .Values.APIOverrides.enabled -}}
```

Reply:
correct suggestion, but i've solved it differently in the end, see comment of mkilchhofer